### DISCUSSION TITLE
Using URL for specPath

### DISCUSSION THREAD

**Discussion by nobrien102** [PaloAltoNetworks/docusaurus-openapi-docs#601] [Q&A]
Hello!
Just started working with docusuarus and this plugin. I've been able to get api docs to be generated from an example URL https://redocly.github.io/redoc/openapi.yaml. I have docs that I would like to try that are living in source control on Azure. They require authorization to be able to fetch them. Is there a way to set a header or token that will make this possible? 

Thank you in advance!

**Comment by jsanguyen** [comment]
Hey! Im running across the same situation with a project I'm building. I wonder if this is possible? 

Bump
